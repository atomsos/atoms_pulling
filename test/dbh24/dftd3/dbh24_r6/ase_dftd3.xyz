7

C       0.377452056683839      1.079180145677113      0.627192351352039
O      -0.034796758058074     -0.322349658388437      0.009805361159987
H      -0.533417553909736      0.777908548524349     -0.281586731033184
H       1.157477853010087     -0.440945979835249      0.522660459567899
H       0.541446334155624      1.535256951767983      1.567870433997500
H       0.805837252069164      1.373967698080599     -0.171866132479246
F       0.453835547579133      0.009581600830846      4.996163135902294
