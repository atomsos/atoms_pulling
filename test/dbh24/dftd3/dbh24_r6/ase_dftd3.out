  _________________________________
                                   
 |         DFTD3 V3.1 Rev 1        |
 | S.Grimme, University Bonn       |
 |         October  2015           |
 |   see dftd3 -h for options      |
  _________________________________

 Please cite DFT-D3 work done with this code as:
 S. Grimme, J. Antony, S. Ehrlich and H. Krieg,
 J. Chem. Phys. 132 (2010), 154104
 If used with BJ-damping cite also
 S. Grimme, S. Ehrlich and L. Goerigk,
 J. Comput. Chem. 32 (2011), 1456-1465
 For DFT-D2 the reference is
 S. Grimme, J. Comput. Chem., 27 (2006), 1787-1799
 For DFT-D3M or DFT-D3M(BJ) the reference is
 D.G.A. Smith, L.A. Burns, K. Patkowski, and 
 C.D. Sherrill, J. Phys. Chem. Lett. 7 (2016) 2197-2203

  files read :     
 ase_dftd3.xyz
C6 coefficients used:
           2  C6 for element            1
Z=  1 CN= 0.912     C6(AA)=    3.03
Z=  1 CN= 0.000     C6(AA)=    7.59
           5  C6 for element            6
Z=  6 CN= 0.000     C6(AA)=   49.11
Z=  6 CN= 0.987     C6(AA)=   43.25
Z=  6 CN= 1.998     C6(AA)=   29.36
Z=  6 CN= 2.999     C6(AA)=   25.78
Z=  6 CN= 3.984     C6(AA)=   18.21
           3  C6 for element            8
Z=  8 CN= 0.000     C6(AA)=   15.51
Z=  8 CN= 0.993     C6(AA)=   12.82
Z=  8 CN= 1.989     C6(AA)=   10.37
           2  C6 for element            9
Z=  9 CN= 0.000     C6(AA)=    9.69
Z=  9 CN= 0.998     C6(AA)=    7.13

#               XYZ [au]               R0(AA) [Ang.]  CN       C6(AA)     C8(AA)   C10(AA) [au] 
   1   0.71328   2.03935   1.18522   c     1.455    3.771        18.9       546.0     19345.1
   2  -0.06576  -0.60915   0.01853   o     1.241    1.900        10.5       211.0      5215.6
   3  -1.00801   1.47003  -0.53212   h     1.091    1.359         3.0        36.6       542.6
   4   2.18732  -0.83327   0.98769   h     1.091    0.455         5.0        60.8       901.0
   5   1.02319   2.90121   2.96285   h     1.091    0.997         3.1        37.4       553.5
   6   1.52281   2.59642  -0.32478   h     1.091    1.006         3.1        37.3       552.7
   7   0.85762   0.01811   9.44138   f     1.150    0.000         9.6       165.0      3457.6

molecular C6(AA) [au] =       323.88

           DFT-D V3
 DF pbe                                               
 parameters
 s6       :    1.0000
 s8       :    0.7220
 rs6      :    1.2170
 rs18     :    1.0000
 alpha6   :   14.0000
 alpha8   :   16.0000
 k1-k3    :   16.0000    1.3333   -4.0000
 Cutoff   :   95.0000 a.u.
 CN-Cutoff:   40.0000 a.u.

 Edisp /kcal,au:    -0.4116 -0.00065589

 E6    /kcal :    -0.1187
 E8    /kcal :    -0.2928
 % E8        : 71.15

 doing analytical gradient O(N^2) ...

 |G|=   2.3179154206605041E-003
gdisp time     0.0
 Cartesian gradient written to file dftd3_gradient
 normal termination of dftd3
